src/Bootstrap/less/theme/page-display-package-v2.less

```
@@ -185,6 +185,47 @@
         margin-top: 3px;
       }
     }
+
+    .sidebar-headers {
+      font-size: 16px;
+      font-weight: 600;
+      color: #666666;
```

This can be `@gray-light`:

```suggestion
      color: @gray-light;
```

---

done 👍